styles/Movie.module.css

```diff
@@ -0,0 +1,73 @@
+.container {
+  padding: 0 2rem;
+}
+
+.main {
+  width: 100%;
+  max-width: 1100px;
+  height: auto;
+  margin: 17px auto;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.main-container {
```

Comment:
By using this style of naming your classes with CSS modules, you limit yourself to using `styles["main-container"]` in your JS code. If you used camelCase (or any other valid JS name style) here, like `mainContainer`, you could do `styles.mainContainer` in your JS instead.